src/store/actions/transactions.test.js

```diff
@@ -63,5 +90,198 @@ describe('actions: transactions', () => {
+
+      await transactionsRetrieved(transactionData)(dispatch, getState);
+      expect(dispatch).toHaveBeenLastCalledWith({
+        type: actionTypes.transactionsRetrieved,
+        data: expectedAction,
+      });
+    });
+
+    it('should dispatch transactionsLoadFailed action if rejected', async () => {
+      const transactionError = new Error('Transaction retrieve error');
+      transactionsApi.getTransactions.mockRejectedValue(transactionError);
+
+      expect(transactionsApi.getTransactions).rejects.toThrow('Transaction retrieve error');
+
+      await transactionsRetrieved(data)(dispatch, getState);
+      expect(transactionsApi.getTransactions).rejects.toThrow(transactionError);
+      expect(dispatch).toHaveBeenCalledTimes(2);
+      expect(dispatch).toHaveBeenNthCalledWith(1, {
+        type: actionTypes.loadingStarted,
+        data: actionTypes.transactionsRetrieved,
+      });
+      expect(dispatch).toHaveBeenNthCalledWith(2, {
+        type: actionTypes.transactionLoadFailed,
+        data: { error: transactionError },
+      });
+    });
+  });
+
+  describe('emptyTransaction', () => {
+    it('should create an action to empty transactions', () => {
+      const expectedAction = { type: actionTypes.emptyTransactionsData };
+      expect(emptyTransactionsData()).toEqual(expectedAction);
+    });
+  });
+
+  describe('pendingTransactionAdded', () => {
+    it('should create an action to add pending transaction', () => {
+      const data = {
+        moduleAssetId: '1938573839:g45krEIjwK',
+        id: '4emF3me9YJSbcIuOp',
+        fee: '1032519n',
+        nonce: '2n',
+        signatures: ['xnVCm30IUhtYidgBX', 'uxsFGiaqS3n4ydB'],
+        sender: {
+          address: '3040783849904107057L',
+          publicKey: 'MIICXAIBAAKBgQCqGKukO1De7zhZj6+H0qtjTkVxwTCpvKe4eCZ0FPqri0cb2JZfXJ/DgYSF6',
+        },
+        isPending: true,
+      };
+      const expectedAction = {
+        type: actionTypes.pendingTransactionAdded,
+        data,
+      };
+      expect(pendingTransactionAdded(data)).toEqual(expectedAction);
+    });
+  });
+
+  describe('resetTransactionResult', () => {
+    it('should create an action to reset transaction result', () => {
+      const expectedAction = { type: actionTypes.resetTransactionResult };
+      expect(resetTransactionResult()).toEqual(expectedAction);
+    });
+  });
+
+  describe('transactionCreated', () => {
+    it('should dispatch transactionCreatedSuccess action if there are no errors', async () => {
+      transactionsApi.create.mockResolvedValue({ tx: newTransaction });
+      const expectedAction = {
+        type: actionTypes.transactionCreatedSuccess,
+        data: { tx: newTransaction },
+      };
+      await transactionCreated(newTransaction)(dispatch, getState);
+      expect(transactionsApi.create).toHaveBeenCalled();
+      expect(dispatch).toHaveBeenCalledWith(expectedAction);
+    });
+
+    it('should dispatch transactionSignError action if there are errors', async () => {
```

Comment:
Please try to apply this format so that we have consistency both for reading and as a general standards

```
// Arrange
const transactionError = new Error('Transaction create error');
loginTypes.passphrase.code = 1;
hwManagerApi.signSendTransaction.mockRejectedValue(transactionError);
const expectedAction = {
        type: actionTypes.transactionSignError,
        data: transactionError,
};

// Act
await transactionCreated(newTransaction)(dispatch, getState);

// Assert
expect(hwManagerApi.signSendTransaction).rejects.toThrow(transactionError);
expect(hwManagerApi.signSendTransaction).toHaveBeenCalled();
expect(dispatch).toHaveBeenCalledWith(expectedAction);
```

Reply:
Fixed